src/shared/utils/gtagEvents.ts

```diff
@@ -194,3 +196,10 @@ export const sendGtagEventCacheUpdate = (
     cache_name: cacheName,
   });
 };
+
+export const sendGtagEventJobSubmit = (
+  job: JobTypes,
+  parameters?: Record<string, string>
+) => {
+  sendGtagEvent('job_submit', parameters);
```

Comment:
I notice now that `job` is not used here, it might be good to add it to the parameters so that we could separate events based on this dimension.
It might be good to call this `job_type` when sending to GA